src/app/dashboard/components/dashboard-under-follow-up/dashboard-under-follow-up.component.html

```diff
@@ -1 +1,70 @@
-<p>dashboard-under-follow-up works!</p>
+<h2>
+  <strong>{{ total }}</strong>
+  {{ 'strings.headingUnderFollowUp' | translate }}
+  <mat-icon
+    [matTooltip]="'descriptions.descDashboardFollowUpInfo' | translate"
+    matTooltipPosition="below"
+    [matTooltipClass]="'follow-up-tooltip'"
```

Comment:
no point in binding since you pass a string

Reply:
modified.